backend/de.metas.printing/de.metas.printing.base/src/main/sql/postgresql/system/80-de.metas.printing/5620680_sys_gh12221_AddBill_userInPQAndImproveFilteringParams.sql

```diff
@@ -0,0 +1,96 @@
+-- 2022-01-06T09:43:57.184254500Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Table_Process SET WEBUI_ViewQuickAction='Y', WEBUI_ViewQuickAction_Default='Y',Updated=TO_TIMESTAMP('2022-01-06 11:43:57','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Table_Process_ID=541029
+;
+
+-- 2022-01-06T09:49:38.493623Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process SET Name='Abrechnung starten',Updated=TO_TIMESTAMP('2022-01-06 11:49:38','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Process_ID=584953
```

Comment:
the processe's displayed help text also needs the information on how to config which PDFs are concatenated (sysconfig).